addons/mail/models/mail_thread.py

```diff
@@ -1833,7 +1833,7 @@ def _message_add_suggested_recipient(self, result, partner=None, email=None, lan
             partner_info = self._message_partner_info_from_emails([email])[0]
             if partner_info.get('partner_id'):
                 partner = self.env['res.partner'].sudo().browse([partner_info['partner_id']])[0]
-        if email and email in [val['email'] for val in result]:  # already existing email -> skip
+        if email and email in [val.get('email') for val in result]:  # already existing email -> skip
```

Comment:
Would avoid to accumulate empty emails, like
```suggestion
        if email and email in [val['email'] for val in result if val.get('email')]:  # already existing email -> skip
```

Reply:
Hello @tde-banana-odoo 

The changes have been made according to your suggestions.

Thank you.